kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/AbstractWatchManager.java

```diff
@@ -368,7 +368,7 @@ protected void onMessage(String message, WatchRequestState state) {
 
           onStatus(status, state);
         } else {
-          logger.error("Error received, but object is not a status - will retry");
+          logger.error("Received an error which is not a status but {} - will retry", object);
```

Comment:
```suggestion
          logger.error("Received an error which is not a status but {} - will retry", message);
```

Reply:
It might also be nice to have a debug / trace log for all messages just in case - I know that the error messages for everything else that is not a Status or HasMetadata does include the message, but it will future proof things a bit more.